concourse/tasks/package-gointegtest.yaml

```diff
@@ -0,0 +1,25 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/gointegtest
+    tag: latest
+
+inputs:
+- name: credentials
+- name: guest-test-infra
+
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: "credentials/credentials.json"
+
+run:
+  path: /daisy
+  args:
+  - -project=gcp-guest
+  - -zone=us-west1-b
+  - -var:repo_name=((repo-name))
+  - -var:repo_owner=GoogleCloudPlatform
+  - -var:git_ref=master
+  - -var:gcs_path=gs://gcp-guest-test-outputs/$(date '+%s')
```

Comment:
this will not work. `$(command)` syntax is enabled by the shell, but we are not using any shell here